src/app/modules/form/admin-form/admin-form.types.ts

```diff
@@ -76,4 +76,16 @@ export type FormUpdateParams = {
   webhook?: IForm['webhook']
 }
 
+export type SettingsUpdateBody = Partial<{
+  authType: NonNullable<IForm['authType']>
+  emails: NonNullable<IForm['emails']>
+  esrvcId: NonNullable<IForm['esrvcId']>
+  hasCaptcha: NonNullable<IForm['hasCaptcha']>
+  inactiveMessage: NonNullable<IForm['inactiveMessage']>
+  status: NonNullable<IForm['status']>
+  submissionLimit: NonNullable<IForm['submissionLimit']>
+  title: NonNullable<IForm['title']>
+  webhook: NonNullable<IForm['webhook']>
+}>
```

Comment:
can we rename this to `SettingsUpdateDTO` to emphasise that it is a _data transfer object_?

also, would it be clearer to define it as `Partial<Pick<IForm, ...>>` instead to emphasise that it is a subset of properties from `IForm`?

Reply:
Resolved, add new `FormSettings` type in `types/form` and `SettingsUpdateDto` type in `shared/typings` in d3afefa